website/source/docs/post-processors/shell-local.html.md

```diff
@@ -19,7 +19,7 @@ some task with packer outputs and variables.
 
 The example below is fully functional.
 
-``` json
```

Comment:
In the same file line 167 I think needs a ```json too

Reply:
good! fixing everything!